packages/stream/airdrop.ts

```diff
@@ -0,0 +1,190 @@
+import { BN, Program, Provider } from "@project-serum/anchor";
+import {
+  Connection,
+  Keypair,
+  PublicKey,
+  SystemProgram,
+  SYSVAR_RENT_PUBKEY,
+} from "@solana/web3.js";
+import {
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  Token,
+  TOKEN_PROGRAM_ID,
+} from "@solana/spl-token";
+import type { Idl } from "@project-serum/anchor/dist/cjs/idl";
+import type { TransactionSignature } from "@solana/web3.js";
+
+import airdrop from "./idl/airdrop";
+import { AIRDROP_TEST_TOKEN } from "./constants";
+import { Wallet } from "@project-serum/anchor";
+
+const PROGRAM_ID = "Ek6Jpdv5iEEDLXTVQ8UFcntms3DT2ewHtzzwH2R5MpvN";
+
+function initProgram(connection: Connection, wallet: Wallet): Program {
+  const provider = new Provider(connection, wallet, {});
+  return new Program(airdrop as Idl, PROGRAM_ID, provider);
+}
+
+export async function initialize(
```

Comment:
since these are exposed, we should add usage examples in readme

Reply:
I used airdrop only for testing to use some spl-tokens. Copied whole logic from streamflow-app and did not change anything. Should we expose them then?